components/Footer/Footer.js

```diff
@@ -3,8 +3,7 @@ import Link from "next/link";
 function Footer() {
   return (
     <footer
-      className="text-white"
-      style={{ background: "linear-gradient(180deg,white 25%,#131249 25%)" }}
+      className="text-white inline-footer-gradient"
```

Comment:
No external CSS Bro! 🥲
Any specific reason for changing that?

We're strictly working with tailwind